sdk/attestation/azure-security-attestation/README.md

```diff
@@ -143,7 +141,7 @@ Each Microsoft Azure Attestation service instance operates in either "AAD" mode
 The Microsoft Azure Attestation service supports attesting different types of evidence depending on the environment.
```

Comment:
nit: This can be just "The Azure Attestation service..."

Reply:
It can't be in the first incarnation, but I've simplified most of the others.